addons/payment_test/tests/common.py

```diff
@@ -0,0 +1,20 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.payment_test.controllers.main import PaymentTestController
+from odoo.addons.payment.tests.common import PaymentCommon
+
+
+class PaymentTestCommon(PaymentCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.test = cls._prepare_acquirer('test', update_values={
+            'payment_icon_ids': [(5, 0, 0)],
+            'capture_manually': False, # Only activate manual capture in dedicated tests
+            'fees_active': False, # Only activate fees in dedicated tests
+        })
+
+        cls.acquirer = cls.test
```

Comment:
After further investigation, it appears that I can't remove it because it sets the provider to 'Test'. If I delete it, no transaction would be found by `_get_tx_from_notification_data` because the transaction would be created for the `dummy` provider. On the other hand, I'll remove the `payment_icon_ids` because it's useless here.

Reply:
If you modify cls.acquirer (without using a cls.test variable), the transactions will be created for the test acquirer no?

Reply:
Nope. Tested and not approved xD The default 'dummy' acquirer provider is `None`

See:
- https://github.com/odoo/odoo/blob/master/addons/payment/tests/common.py#L89
- https://github.com/odoo/odoo/blob/master/addons/payment/tests/common.py#L135

Reply:
Pushed what I meant :P